### PR TITLE
[zh-cn]sync kubeadm-reconfigure topology-manager configure-multiple-schedulers onfigure-liveness-readiness

### PR DESCRIPTION
**Pull request by my-git9**
```
# Changes to be committed:
#       modified:   content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md
#       modified:   content/zh-cn/docs/tasks/administer-cluster/topology-manager.md
#       modified:   content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md
#       modified:   content/zh-cn/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md

```

### CODE REVIEW

File: content/zh-cn/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 1]
```diff
@@ -82,15 +82,24 @@ Save the file as `Dockerfile`, build the image and push it to a registry. This e
...
 For more details, please read the GCR
-[documentation](https://cloud.google.com/container-registry/docs/).
+[documentation](https://cloud.google.com/container-registry/docs/). Alternatively
+you can also use the [docker hub](https://hub.docker.com/search?q=). For more details
+refer to the docker hub [documentation](https://docs.docker.com/docker-hub/repos/create/#create-a-repository).
 -->
 将文件保存为 `Dockerfile`，构建镜像并将其推送到镜像仓库。
 此示例将镜像推送到 [Google 容器镜像仓库（GCR）](https://cloud.google.com/container-registry/)。
 有关详细信息，请阅读 GCR [文档](https://cloud.google.com/container-registry/docs/)。
+或者，你也可以使用 [docker hub](https://hub.docker.com/search?q=)。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:92]:
```suggestion
或者，你也可以使用 [Docker Hub](https://hub.docker.com/search?q=)。
```

File: content/zh-cn/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 2]
```diff
@@ -82,15 +82,24 @@ Save the file as `Dockerfile`, build the image and push it to a registry. This e
...
+有关更多详细信息，请参阅 docker hub
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:93]:
```suggestion
有关更多详细信息，请参阅 Docker Hub
```

File: content/zh-cn/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 3]
```diff
@@ -82,15 +82,24 @@ Save the file as `Dockerfile`, build the image and push it to a registry. This e
...
 
+<!--
+# The image name and the repository
+# used in here is just an example
+-->
 ```shell
-docker build -t gcr.io/my-gcp-project/my-kube-scheduler:1.0 .
-gcloud docker -- push gcr.io/my-gcp-project/my-kube-scheduler:1.0
+docker build -t gcr.io/my-gcp-project/my-kube-scheduler:1.0 .     # 镜像名称和仓库
+gcloud docker -- push gcr.io/my-gcp-project/my-kube-scheduler:1.0 # 这里使用的只是一个例子
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:102]:
```suggestion
# 这里使用的镜像名称和仓库只是一个例子
docker build -t gcr.io/my-gcp-project/my-kube-scheduler:1.0 .
gcloud docker -- push gcr.io/my-gcp-project/my-kube-scheduler:1.0
```

**Comment by windsonsea** [comment]
Thanks.
/approve

**Pull request merged**